src/main.rs

```diff
@@ -1649,6 +1657,76 @@ fn cmd_regenerate_audit_as(
     Ok(())
 }
 
+fn cmd_renew(out: &Arc<dyn Out>, cfg: &Config, sub_args: &RenewArgs) -> Result<(), miette::Report> {
+    trace!("renewing wildcard audits");
+    let mut store = Store::acquire_offline(cfg)?;
+    do_cmd_renew(out, &mut store, sub_args);
+    store.commit()?;
+    Ok(())
+}
+
+fn do_cmd_renew(out: &Arc<dyn Out>, store: &mut Store, sub_args: &RenewArgs) {
+    assert!(sub_args.expiring ^ sub_args.crate_name.is_some());
+
+    let today = <chrono::DateTime<chrono::Utc>>::from(SystemTime::now()).date_naive();
```

Comment:
The point of these `do_...` methods is generally to allow testing, which means that we shouldn't be pulling system state like the timestamp within `do_cmd_renew` - could you make this an argument so it can be a fixed date in tests?

Reply:
Will do. I had a similar realization (though the tests I added aren't impacted by different dates). It'd be better to make it a parameter.

Reply:
Might be nice to also capture a diff of the store in the tests as well, which would show the dates.